src/cameras/map/marker/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import LocationOnIcon from '@material-ui/icons/LocationOn'
+
+import { useStyles } from 'cameras/map/style'
+import { Info } from 'cameras/map/marker/info'
+
+export const Marker = ({ item }) => <>
+  <div className={useStyles().markerPosition}>
+    <LocationOnIcon fontSize={'large'} />
+  </div>
+  {item.show && <Info item={item} />}
+</>
+
+Marker.propTypes = { item: PropTypes.object.isRequired }
```

Comment:
Whilst `PropTypes.object` is technically correct, wouldn't it be more helpful to be more specific? An item's shape would be, if I'm not mistaken, the following:
```ts
type Item = {
    id: string,
    longitude: double,
    latitude: double,
    show: boolean
```

The equivalent in prop-types would be:
```js
Marker.propTypes = {
    item: PropTypes.exact({
        id: PropTypes.string,
        longitude: PropTypes.number,
        latitude: PropTypes.number,
        show: PropTypes.bool
    })
```

This signature can be extracted to a separate file and used everywhere it is required.

Reply:
Done.